README.md

```diff
@@ -55,6 +60,98 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+## Adding remote repositories
+
+Notes:
+
+- For devfile hosts (environment variable or config file) specify a remote url or a local folder structure and `index.json` that follows the `devfile/api` spec.
+- You can have infinitely many sources as long as the name for the source repository is different.
```

Comment:
I dont think we should have this restriction. We should err out if there is a dup key from either sources. But we can work on this later.

Reply:
I handle it this way for simplicity sake and because searching for a `json` file could take a lot longer than just specifying `index.json`